### PR TITLE
 MEB-100: Add a new datasets page

### PR DESCRIPTION
**Pull request by PROTechThor**
The following changes have been made: 

1. Created a two column layout
2. Made the logos a bit smaller
3. Made the text flow around the logos
4. Streamlined the headings -- there is a lot of duplication -- not each heading needs to have the "Data Dumps" in.
5. Streamlined the descriptions 
6. Added a nice catch all commercial use statement at the top, with basically this sentiment: "we ask commercial users to support us in order to help fund the creation and maintenance of these datasets."

### CODE REVIEW

**Comment by PROTechThor** [comment]
@mayhem @paramsingh Any thoughts/suggestions/issues?

**Comment by paramsingh** [comment]
@brainzbot add to whitelist

**Comment by paramsingh** [comment]
Do you mind posting a screenshot for an easier review?

**Comment by PROTechThor** [comment]
@paramsingh A screenshot of the output?

**Comment by paramsingh** [comment]
A screenshot of the page in the browser

**Comment by PROTechThor** [comment]
@paramsingh I was having trouble setting up the whole repo locally (because Docker was failing), so I skipped that part. So I simply tested it using a codepen pen with only the actual html/css files. The entire look doesn't have the MB styles, but you can see the changed texts and the 2 column layout, as well as the smaller logos and text flowing around the logo. So I believe the intended changes have been applied?
capture

File: metabrainz/templates/index/datasets.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ _('Datasets') }} - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+  <h1 class="page-title">{{ _('The MetaBrainz Datasets') }}</h1>
+
+  <p> We ask <a href="{{ url_for('users.account_type') }}">commercial users</a> to <a href="{{ url_for('users.account_type') }}">support us</a> in order to help fund the creation and maintenance of these datasets. </p>
```
Comment (paramsingh) [inline metabrainz/templates/index/datasets.html]: This should be part of the next paragraph that starts `The MetaBrainz Foundataion provides free, open access...`

File: metabrainz/templates/index/datasets.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+<p>
+  The MetaBrainz Foundation provides free, open access to various types of useful data gathered and verified by volunteers. Here, we list and describe the various datasets we maintain.
```
Comment (paramsingh) [inline metabrainz/templates/index/datasets.html]: I would prefer it if all the text in the html is not in one line, makes it easier to read in text editors and in the github review interface.

Comment (paramsingh) [reply metabrainz/templates/index/datasets.html]: This goes for all descriptions. We generally try to keep lines around a 100-120 columns long.

**Comment by PROTechThor** [comment]
@paramsingh Anything else?

**Pull request merged**